### DISCUSSION TITLE
Extract generated snapshot URL

### DISCUSSION THREAD

**Discussion by zerquanta** [koush/scrypted#1287] [Q&A]
Hi,

I'm using EZVIZ cameras connected to Scrypted via RTSP. When accessing the camera through Scrypted devices section, I can view the snapshot of the selected camera generated by Scrypted. My goal is to extract the URL of this snapshot so that I can use it in other applications (Node-RED, etc.), similar to how it is done with RTSP rebroadcasting. It can be done?

Thanks

**Comment by koush** [comment]
web hook plugin
